src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String fruitName;
+    private int quantity;
+    private int sold;
```

Comment:
Why do you need information about how much fruits sold? I think it is enough just to check how much left in db, to proceed with all oprerations